### ISSUE TITLE
Multiple message-header fields

### ISSUE THREAD

**Issue by wilzbach** [storyscript/platform-engine#415]
https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2

> Multiple message-header fields with the same field-name MAY be present in a message if and only if the entire field-value for that header field is defined as a comma-separated list [i.e., #(values)]. It MUST be possible to combine the multiple header fields into one "field-name: field-value" pair, without changing the semantics of the message, by appending each subsequent field-value to the first, each separated by a comma. The order in which header fields with the same field-name are received is therefore significant to the interpretation of the combined field value, and thus a proxy MUST NOT change the order of these field values when a message is forwarded.

This clearly isn't implemented:

https://github.com/storyscript/platform-engine/blob/2570b8d11799ce5b17730b26d3fa8d0e7ca7f835/asyncy/http_handlers/StoryEventHandler.py#L108-L111

https://2.python-requests.org/en/master/_modules/requests/structures

We could ignore it as it's not an important aspect, though then it would need to be documented.

**Comment by judepereira** [comment]
@wilzbach I don't think this is a problem - upstream (load balancer/gateway) should've concatenated the field value properly, since by the time it reaches the engine, it's in the form of a map, and as such, keys can only be unique here.
